### PR TITLE
Light the beacons.

### PR DESCRIPTION
**Pull request by petemounce**

### CODE REVIEW

File: clang.nuspec.template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <metadata>
+    <id>clang</id>
+    <version>$($tvVersion)</version>
+    <packageSourceUrl>https://github.com/bimmlerd/clang-choco</packageSourceUrl>
+    <owners>bimmlerd, petemounce</owners>
+
+    <title>LLVM compiler infrastructure clang (Install)</title>
+    <authors>LLVM team</authors>
+    <projectUrl>http://clang.llvm.org/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/clang.png</iconUrl>-->
```
Comment (bimmlerd) [inline clang.nuspec.template:12]: replace your repo? :D

File: release.ps1 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+You need to:
+.\build.ps1
+.\test.ps1
+
+retry release.ps1
+"@
+}
+
+& choco push $package
```
Comment (bimmlerd) [inline release.ps1:25]: maybe check for error here and print error info if not successful?

File: test.ps1 [hunk 3]
```diff
@@ -0,0 +1,40 @@
+param(
+  [switch] $prerelease
+)
+
+choco uninstall clang --force -y
+if ($prerelease) {
+  choco install clang --verbose --debug --prerelease --force -y -s ".;https://chocolatey.org/api/v2/"
```
Comment (bimmlerd) [inline test.ps1]: what does the `;https://chocolatey...` do again?

**Review by bimmlerd — CHANGES_REQUESTED** [review]
minor nits, LGTM otherwise

**Comment by bimmlerd** [comment]
Oh and could you also remove the default README and replace it with a brief sensible one?

File: clang.nuspec.template (hunk 1 above)
Comment (petemounce) [reply clang.nuspec.template:12]: I don't have an icon URL at all, and it's commented as a reminder to get one. Was planning to ask the LLVM team for a PR to fix it if they care to.

File: release.ps1 (hunk 2 above)
Comment (petemounce) [reply release.ps1:25]: 👍

File: test.ps1 (hunk 3 above)
Comment (petemounce) [reply test.ps1]: The --sources param takes a CSV list of sources to choose from. Here, we don't want to take a remote source, so I'll remove that.

**Comment by petemounce** [comment]
I think I've done everything there, but don't know how the GH reviewing works yet.

**Review by bimmlerd — APPROVED** [review]
LGTM

**Pull request merged**